### PR TITLE
Create CITATION.cff

### PR DESCRIPTION
**Pull request by Zack-83**

### CODE REVIEW

**Comment by jochenklar** [comment]
Thanks @Zack-83 , I guess we discuss this in the next steering group meeting. One technical point: I think CFF does not have `contributors`, I think they should be part of the authors list. https://citation-file-format.github.io/assets/pdf/cff-specifications-1.0.3.pdf

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    given-names:  Džulia
+    orcid:        https://orcid.org/0000-0002-1698-6826
+  - family-names: Wallace
+    given-names:  David
+    orcid:        https://orcid.org/0000-0001-8958-4601
+  - family-names: Windeck
+    given-names:  Jürgen
+    orcid:        https://orcid.org/0000-0003-1909-4353
+
+contributors:
```
Comment (Zack-83) [inline CITATION.cff:80]:
```suggestion
```

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    orcid:        https://orcid.org/0000-0003-3847-2663
+  - family-names: Martínez Muñoz
+    given-names:  David
+    orcid:        
+  - family-names: Lanza
+    given-names:  Margherita
+    orcid:        
+  - family-names: Pilori
+    given-names:  Dario
+    orcid:        https://orcid.org/0000-0003-4499-8573
```
Comment (Zack-83) [inline CITATION.cff:102]:
```suggestion
  - family-names: Cardenas
    given-names:  Yonny
    orcid:        
  - family-names: Grasso Toro
    given-names:  Federico
    orcid:        https://orcid.org/0000-0002-9041-0868
  - family-names: Lanza
    given-names:  Margherita
    orcid:        
  - family-names: Linares
    given-names:  Jimena
    orcid:        https://orcid.org/0000-0003-3847-2663
  - family-names: Marmin
    given-names:  Sébastien
    orcid:        
  - family-names: Martínez Muñoz
    given-names:  David
    orcid:        
  - family-names: Piette
    given-names:  Anne-Sophie
    orcid:        https://orcid.org/0000-0002-7876-1251
  - family-names: Pilori
    given-names:  Dario
    orcid:        https://orcid.org/0000-0003-4499-8573
```

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    orcid:        https://orcid.org/0000-0001-9164-3500
+  - family-names: Hendricks
+    given-names:  Sonja
+    orcid:        
+  - family-names: Höhner
+    given-names:  Kathrin
+    orcid:        https://orcid.org/0000-0002-3988-7839
+  - family-names: Lanza
+    given-names:  Giacomo
+    orcid:        https://orcid.org/0000-0002-2239-3955
```
Comment (Zack-83) [inline CITATION.cff:30]:
```suggestion
  - family-names: Klar
    given-names:  Jochen
    orcid:        https://orcid.org/0000-0002-5883-4273
  - family-names: Lanza
    given-names:  Giacomo
    orcid:        https://orcid.org/0000-0002-2239-3955
```

**Comment by jochenklar** [comment]
Hi @Zack-83 , I would suggest that we add Claudia Engelhardt and Janine Straka (@leucoryx), since they created the main RDMO content during the DFG Projects.

File: CITATION.cff [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+message: If you refer to this content in a publication, please cite it as below.
+title: Content for Research Data Management Organizer (RDMO)
+doi: 10.5281/zenodo.596706
+license: CC0 1.0 Universal
+repository-code: https://github.com/rdmorganiser/rdmo-catalog
+
+authors:
+  - family-names: Frenzel
+    given-names:  Johannes
+    orcid:        https://orcid.org/0000-0003-0458-5424
```
Comment (Zack-83) [inline CITATION.cff:12]:
```suggestion
  - family-names: Engelhardt
    given-names:  Claudia
    orcid:        https://orcid.org/0000-0002-3391-7638
  - family-names: Frenzel
    given-names:  Johannes
    orcid:        https://orcid.org/0000-0003-0458-5424
```

File: CITATION.cff [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+    orcid:        
+  - family-names: Sinn
+    given-names:  Christoph
+    orcid:        https://orcid.org/0000-0003-0275-1070
+  - family-names: Spenger
+    given-names:  Martin
+    orcid:        https://orcid.org/0000-0002-8841-5985
+  - family-names: Sutter
+    given-names:  Paul
+    orcid:        https://orcid.org/0000-0002-9883-0680
```
Comment (Zack-83) [inline CITATION.cff:72]:
```suggestion
  - family-names: Straka
    given-names:  Janine
    orcid:        https://orcid.org/0000-0002-0695-1689
  - family-names: Sutter
    given-names:  Paul
    orcid:        https://orcid.org/0000-0002-9883-0680
```

**Comment by Zack-83** [comment]
> Hi @Zack-83 , I would suggest that we add Claudia Engelhardt and Janine Straka (@leucoryx), since they created the main RDMO content during the DFG Projects.

Done https://github.com/rdmorganiser/rdmo-catalog/pull/190/commits/3c1cd8afbf6a046174faf5ef6794cd2e0d9b5187

**Comment by Zack-83** [comment]
Laut GH waren aktiv ab 2022:




io
Olaf
Jochen
Yves
Torsten
Paul
Federico
Laura M
Sabine
Alexander
Laura R
Jürgen W
Thomas R
Johannes F
Clara
Robin
Martin
Christoph
Tim
Dario

ehemals:

Janine
Harry
Thomas S

**Comment by Zack-83** [comment]
Hallo allerseits! Ich habe die Autorenreihenfolge nach Gefühl geändert. Bitte gebt Bescheid bis Donnerstag 28.09., falls ihr nach oben (oder nach unten) verschoben werden wollt!

**Review by yvgrossmann — COMMENTED** [review]
super 👍 thx @Zack-83

File: CITATION.cff [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+  - family-names: Frenzel
+    given-names:  Johannes
+    orcid:        https://orcid.org/0000-0003-0458-5424
+  - family-names: Schröder
+    given-names:  Clara
+    orcid:        
+  - family-names: Spenger
+    given-names:  Martin
+    orcid:        https://orcid.org/0000-0002-8841-5985
+  - family-names: Terzijska
```
Comment (Zack-83) [inline CITATION.cff:55]:
```suggestion
  - family-names: Spenger
    given-names:  Martin
    orcid:        https://orcid.org/0000-0002-8841-5985
  - family-names: Wettermann
    given-names:  Sarah
    orcid:        https://orcid.org/0009-0006-5838-095X
  - family-names: Terzijska
```

File: CITATION.cff [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+    orcid:        https://orcid.org/0000-0002-8841-5985
+  - family-names: Wettermann
+    given-names:  Sarah
+    orcid:        https://orcid.org/0009-0006-5838-095X
+  - family-names: Terzijska
+    given-names:  Džulia
+    orcid:        https://orcid.org/0000-0002-1698-6826
+  - family-names: Wallace
+    given-names:  David
+    orcid:        https://orcid.org/0000-0001-8958-4601
```
Comment (Zack-83) [inline CITATION.cff:63]:
```suggestion
  - family-names: Terzijska
    given-names:  Džulia
    orcid:        https://orcid.org/0000-0002-1698-6826
  - family-names: Wallace
    given-names:  David
    orcid:        https://orcid.org/0000-0001-8958-4601
  - family-names: Spenger
    given-names:  Martin
    orcid:        https://orcid.org/0000-0002-8841-5985
  - family-names: Wettermann
    given-names:  Sarah
    orcid:        https://orcid.org/0009-0006-5838-095X
```

File: README.md [hunk 8]
```diff
@@ -28,6 +28,7 @@ For problems or bugs with the RDMO Software, please use the issues in the [rdmo]
 | DCC checklist                                   | [RDMO team](rdmorganiser/questions/dcc.xml)                                                                                            | 10/2018     |
 | Software Management Plan                        | [RDMO team](rdmorganiser/questions/SMP-Questions.xml)                                                                                  | 06/2023     |
 
+
```
Comment (Zack-83) [inline README.md:31]:
```suggestion
```

**Comment by yvgrossmann** [comment]
Thanks @Zack-83 looks 👌

**Review by yvgrossmann — APPROVED** [review]

**Pull request merged**